docs/specs/clients/sign/rpc-methods.md

```diff
@@ -338,3 +342,87 @@ true
 | TTL     | 30       |
 | Tag     | 1115     |
 ```
+
+## Wallet Authentication RPC Methods
+
+### Definitions
+
+- **Nullables:** Fields flagged as `Optional` can be omitted from the payload.
+- Unless explicitly mentioned that a response requires associated data, all methods response's follow a default JSON-RPC pattern for the success and failure cases:
+
+```jsonc
+// Success
+result: true
+
+// Failure
+error: {
+  "code": number,
+  "message": string
+}
+```
```

Comment:
We already have definitions at the top of the file. I think it's redundant to have them twice